automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -214,7 +214,7 @@ def product_upgrade(product):
                                 upgraded = str(execute(
                                     get_sat_cap_version, 'cap', host=cap_host
                                     )[cap_host])
-                                if current:
+                                if current != 'None':
```

Comment:
@elyezer , We dont want to rasie an exception, giving an example on Capsule 6.1 and less versions it is just imposssible to get version. But in 6.2 abd above both sat and cap versions are accessible.